backend-tests/docker/docker-compose.backend-tests.yml

```diff
@@ -10,7 +10,7 @@ services:
             - /var/run/docker.sock:/var/run/docker.sock
         command: ${PYTEST_ARGS}
         environment:
-            PYTHONPATH: "/testutils"
+            PYTHONPATH: "/"
```

Comment:
As far as I know, the whole `backend-tests` can be removed. I will ask to the Server team and follow-up independently (trying to keep the scope creep at bay with your work).